docs/dataloader.adoc

```diff
@@ -0,0 +1,59 @@
+= fastai.dataloader
+
+== Introduction and overview
+
+*Note:* the fastai DataLoader has a similar API to the PyTorch DataLoader. Please see http://pytorch.org/docs/master/data.html#torch.utils.data.DataLoader[the PyTorch documentation] for usage and details. The documentation presented here focuses on the differences between the two classes.
+
+== {{class DataLoader, dataset, batch_size=1, shuffle=False, sampler=None, batch_sampler=None, pad_idx=0, num_workers=None, pin_memory=False, drop_last=False, pre_pad=True, half=False, transpose=False, transpose_y=False}}
+
+.Used to iterate through a dataset to pass data into a model for training.
+
+=== {{arguments}}
+
+For information on arguments with no descriptions, please see http://pytorch.org/docs/master/data.html#torch.utils.data.DataLoader[the PyTorch documentation]
+
+dataset::
+
+batch_size::
+
+shuffle::
+
+sampler::
+
+batch_sampler::
+
+pad_idx (type int, default 0)::
+    A padding index representing how many zeros to add to each batch. See: `pre_pad`
+
+num_workers (type int, default `None`)::
```

Comment:
The arguments should be a template, eg {{arg pad_idx,int,None}}. Not sure if that template has been created yet or not...